jobs/common/pre-reqs.yaml

```diff
@@ -33,12 +33,12 @@
             {packages}
 
 - builder:
-    name: go-get
+    name: go-install
     builders:
       - shell: |-
           #!/bin/bash
           set -eux
-          GO111MODULE=off go get {modules}
+          go install {modules}
```

Comment:
This is the only one I don't know about. All those listed would have to include the specific version/latest.

Reply:
I've changed all the usages of this to be latest